WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1093,7 +1093,7 @@ public int getCount() {
     public static final String NEW_MEDIA_GALLERY = "NEW_MEDIA_GALLERY";
```

Comment:
If we drop `ActivityLauncher.newGalleryPost()`, we can also drop `NEW_MEDIA_GALLERY`, `NEW_MEDIA_GALLERY_EXTRA_IDS`, and `EditPostActivity.prepareMediaGallery()`.